### ISSUE TITLE
[469b] Need make proper high-res textures for stock LookAndFeel classes

### ISSUE THREAD

**Issue by SeriousBuggie** [OldUnreal/UnrealTournamentPatches#898]
Currently hi-res textures just upscaled small textures. UMenu package:
![img](https://user-images.githubusercontent.com/70026933/175545519-40d0c80a-271c-4e20-bb96-c5bdcbab9d66.png)
![img](https://user-images.githubusercontent.com/70026933/175545653-3edd1078-84cd-4ceb-bc5b-235622485656.png)
![img](https://user-images.githubusercontent.com/70026933/175545716-01cfffed-e11f-4ea2-9a12-baed21b9d6e9.png)

**Comment by stijn-volckaert** [comment]
I agree they could be improved. If anyone feels inclined to update these textures, I'd gladly accept the pull request

**Comment by SeriousBuggie** [comment]
I can try use different AI Upscaler.
Current:
![BlueActiveFrameHiRes](https://user-images.githubusercontent.com/70026933/180850598-cd8c7e6b-160d-4ea1-a018-812c1b47a21b.png)
Possible variant via https://github.com/Araxeus/PNG-Upscale
![BlueActiveFrame(EDSRx4)(EDSRx2)](https://user-images.githubusercontent.com/70026933/180850660-0d8ad171-cd9a-4515-80e7-a634b3294367.png)

**Comment by SeriousBuggie** [comment]
Little better version (in source red replace to transparent before upscale):
![BlueActiveFrame(EDSRx4)(EDSRx2)](https://user-images.githubusercontent.com/70026933/180856774-8a1c7af7-9da3-4f33-a401-dd0f7515975a.png)

**Comment by SeriousBuggie** [comment]
Some other experiments:
![BlueActiveFrame_x4x2](https://user-images.githubusercontent.com/70026933/180874510-cafb86e2-f25f-4fc5-9fcd-982219466e63.png)
![BlueActiveFrame_x4](https://user-images.githubusercontent.com/70026933/180874516-f6ab3b7f-6f73-42f5-93d3-7cba8f73394d.png)

**Comment by lafoxxx92** [comment]
Please, take a look and tell if these ones look good enough so I could continue

![Image](https://github.com/user-attachments/assets/3ff0709c-3c7b-41a8-a074-64be5ff18344)

**Comment by lafoxxx92** [comment]
Hi.
?

**Comment by SeriousBuggie** [comment]
Hires only use for hires. It not replace lowres. So there no need look at downscaled version.

**Comment by lafoxxx92** [comment]
Hi,
I'm sorry, but these "new" hi-res files still look terrible (just slightly better than what they used to be).
Lots and lots of artefacts, not pixel-perfect. No idea how they manage to get messed up, but whoever done them (props though! it's still lot of work) clearly didn't playtest to ensure pixel-perfect precision.

<img width="159" height="145" alt="Image" src="https://github.com/user-attachments/assets/165d084f-0642-431d-bfa5-992c96b8a228" />

<img width="328" height="105" alt="Image" src="https://github.com/user-attachments/assets/812ef360-f3ae-4b68-91d5-4d727ad68c4e" />

<img width="201" height="53" alt="Image" src="https://github.com/user-attachments/assets/4db8d851-87d1-4ca8-8bdb-09ee58c1b7d4" />

<img width="198" height="204" alt="Image" src="https://github.com/user-attachments/assets/a0e3ea4b-e061-4616-a67e-9c40d441a1c2" />

<img width="239" height="50" alt="Image" src="https://github.com/user-attachments/assets/0d72227f-c6fe-42b7-aef2-780fc1c691d7" />

<img width="70" height="64" alt="Image" src="https://github.com/user-attachments/assets/de2cccbb-9915-4b2a-a75e-f55d603aa8eb" />

Maybe leave as is (stock low-res)? I'd choose crisp over crooked any time.
